modules/ugdk-core/src/graphic/rendertarget.cc

```diff
@@ -32,13 +34,13 @@ void RenderTarget::Clear(structure::Color color) {
 }
 
 void RenderTarget::Bind() {
-    system::AssertCondition<system::InvalidOperation>(!IsActive(), "RenderTarget must be inactive for Bind.");
+    //system::AssertCondition<system::InvalidOperation>(!IsActive(), "RenderTarget must be inactive for Bind.");
     is_bound_ = true;
     UpdateViewport();
 }
 
 void RenderTarget::Unbind() {
-    system::AssertCondition<system::InvalidOperation>(IsActive(), "RenderTarget must be active for Unbind.");
+    //system::AssertCondition<system::InvalidOperation>(IsActive(), "RenderTarget must be active for Unbind.");
```

Comment:
Are you sure it's ok to remove this assertion?

Reply:
Yes, nowhere else does the code, or the examples for that matter, check or rely on it being active

Reply:
It's okay to unbind things that are bound, because when ~Canvas is called, it tries to unbind things.
I kept the assert on Bind() so things are still safe, the assert on Unbind isn't necessary

Reply:
Then just remove the line